folly/memory/Malloc.h

```diff
@@ -95,8 +95,8 @@ mallctlbymib(const size_t*, size_t, void*, size_t*, void*, size_t)
 
 // for malloc_usable_size
 // NOTE: FreeBSD 9 doesn't have malloc.h.  Its definitions
-// are found in stdlib.h.
-#if __has_include(<malloc.h>)
+// are found in stdlib.h and it errors when including malloc.h.
+#if __has_include(<malloc.h>) && (!defined(__STDC__) || !__STDC__)
```

Comment:
This (or the change to the portability malloc below) is causing `malloc.h` to no longer get included in our internal builds :( 

If these two changes are dropped, the rest should be safe to land :)